velodyne_driver/package.xml

```diff
@@ -1,39 +1,52 @@
 <?xml version="1.0"?>
 <package format="2">
-  <name>velodyne_driver</name>
-  <version>1.5.2</version>
-  <description>
-    ROS device driver for Velodyne 3D LIDARs.
-  </description>
-  <maintainer email="jwhitley@autonomoustuff.com">Josh Whitley</maintainer>
-  <maintainer email="brice.rebsamen@gmail.com">Brice Rebsamen</maintainer>
-  <author>Jack O'Quin</author>
-  <author>Patrick Beeson</author>
-  <author>Michael Quinlan</author>
-  <author>Yaxin Liu </author>
-  <license>BSD</license>
+    <name>velodyne_driver</name>
+    <version>1.5.2</version>
+    <description>
+        ROS device driver for Velodyne 3D LIDARs.
+    </description>
+    <maintainer email="jwhitley@autonomoustuff.com">Josh Whitley</maintainer>
+    <maintainer email="brice.rebsamen@gmail.com">Brice Rebsamen</maintainer>
+    <author>Jack O'Quin</author>
+    <author>Patrick Beeson</author>
+    <author>Michael Quinlan</author>
+    <author>Yaxin Liu </author>
+    <license>BSD</license>
+    
+    <url type="website">http://www.ros.org/wiki/velodyne_driver</url>
+    <url type="repository">https://github.com/ros-drivers/velodyne</url>
+    <url type="bugtracker">https://github.com/ros-drivers/velodyne/issues</url>
 
-  <url type="website">http://www.ros.org/wiki/velodyne_driver</url>
-  <url type="repository">https://github.com/ros-drivers/velodyne</url>
-  <url type="bugtracker">https://github.com/ros-drivers/velodyne/issues</url>
-
-  <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>roslint</build_depend>
-
-  <depend>diagnostic_updater</depend>
-  <depend>dynamic_reconfigure</depend>
-  <depend>libpcap</depend>
-  <depend>nodelet</depend>
-  <depend>pluginlib</depend>
-  <depend>roscpp</depend>
-  <depend>tf</depend>
-  <depend>velodyne_msgs</depend>
+    
+    <buildtool_depend>ament_cmake</buildtool_depend>
+    
+    <build_depend>rclcpp</build_depend>
```

Comment:
Instead of repeating `<build_depend>` and `<exec_depend>`, you can use `<depend>` to do both.